ding_tabroll.features.field_base.inc

```diff
@@ -10,6 +10,37 @@
 function ding_tabroll_field_default_field_bases() {
   $field_bases = array();
 
+  // Exported field_base: 'field_ding_menu_ext'
+  $field_bases['field_ding_menu_ext'] = array(
```

Comment:
All fields should be prefix with the name of the module that defines them. Here it should be ding_tabroll not ding_menu!